zeebe/clients/zeebe-client-spring/src/main/java/io/camunda/zeebe/spring/client/annotation/customizer/ZeebeWorkerValueCustomizer.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.spring.client.annotation.customizer;
+
+import io.camunda.zeebe.spring.client.annotation.value.ZeebeWorkerValue;
+
+/**
+ * This interface could be used to customize the {@link
+ * io.camunda.zeebe.spring.client.annotation.JobWorker} annotation's values. Just implement it and
+ * put it into to the {@link org.springframework.context.ApplicationContext} to make it work. But be
+ * careful: these customizers are applied sequentially and if you need to change the order of these
+ * customizers use the {@link org.springframework.core.annotation.Order} annotation or the {@link
+ * org.springframework.core.Ordered} interface.
+ *
+ * @see io.camunda.zeebe.spring.client.properties.PropertyBasedZeebeWorkerValueCustomizer
```

Comment:
💭  as this references classes from the starter, I'm more in favor of merging this module to the starter, but we can do that in a follow-up